src/module_loader.rs

```diff
@@ -126,24 +170,28 @@ impl ModuleLoader for RustyLoader {
             }
 
             _ => {
+                #[cfg(feature = "custom_import")]
+                // Custom imports can handle any URL scheme
+                return Ok(url);

```

Comment:
Resolve is what decides if the url is ok or not

It's the traits job to ok or deny

Reply:
The trait can deny by returning Err(something), in which case the match statement won't be reached.